cfme/v2v/migrations.py

```diff
@@ -658,12 +670,17 @@ def plan_in_progress(self, vms_count=5):
 
 
 @attr.s
-class InfrastructureMapping(BaseEntity):
+class InfrastructureMapping(BaseEntity, Updateable):
     """Class representing v2v infrastructure mappings"""
     name = attr.ib()
     description = attr.ib(default=None)
     form_data = attr.ib(default=None)
 
+    def update(self, updates):
+        view = navigate_to(self, 'Edit', wait_for_view=20)
+        assert view.wait_displayed()
```

Comment:
`wait_for_view` explicitly waited for 20s above, why additionally `wait_displayed` here on the same view? Is there some issue with the view getting displayed in a reasonable time? Do we need to trigger `is_displayed` on more than just the title appearing?